sprint8/ex1.js

```diff
@@ -0,0 +1,16 @@
+// 1. Tipăriți toate numerele pare de la 0 la 10
+// Încercați să rezolvați această problemă cât mai eficient posibil. Luați în considerare utilizarea buclelor care vă vor permite să tastați cele mai puține caractere / comenzi (se acceptă orice abordare bazată pe bucle). Deși ați putea pur și simplu să tipăriți numerele pare, fiți creativi și încercați să le generați într-un mod în care să funcționeze indiferent de limita setată (până la 10 sau chiar până la 10.000).
+
+// Exemplu 1:
+// Input: n = 10, unde n este numărul maxim până la care se va executa bucla
+// Output: 0, 2, 4, 6, 8, 10
+
+function afisareNrPare(n) {
+  let NrPar = [];
```

Comment:
```suggestion
  let nrPar = [];
```

snake_case
camelCase